src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -398,6 +381,26 @@ export default {
       }
     },
 
+    loadCorrection (filing, section) {
+      if (section) {
+        const item = {
+          type: filing.header.name,
+          filingAuthor: filing.header.certifiedBy,
+          filingDate: filing.correction.correctedFilingDate,
+          filingId: filing.header.filingId,
+          corrFilingId: filing.correction.correctedFilingId,
+          correctedFilingType: this.typeToTitle(filing.correction.correctedFilingType),
+          title: `Correction - ${this.typeToTitle(filing.correction.correctedFilingType)}`,
+          draftTitle: `Correction Filing`,
```

Comment:
Would using typeToTitle() be too messy for either lines 393 or 394?

Where is Draft Title being used? Is this in the mockups? What about the Priority Correction title?

Reply:
Not sure what you mean?                                                                                      
I'm not sure, yes i pulled this down from the example data. If it isn't something we use, then we can certainly do away with it.  The Priority Correction title was used in the todo lists as per design but was a little different for a correction in the filing history list.

Reply:
1. I wonder if typeToTitle() could be used to derive the title here?
2. Oops, you're right -- Priority Correction is in the Todo List. Although maybe ask Scott if it should be in the Filing History List as well. I dunno.

Reply:
I'm not sure what you have planned for draftTitle, but for the main title I thought you might reuse typeToTitle() -- with an appropriate parameter (similar to AR with agmYear).
You can leave this for now and I can show you what I mean later.

Reply:
Ok sounds good, i'm going to remove the Draft title property now until we have a need for it as well.